projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -76,3 +63,22 @@ export default class RunJobDialog extends Component<IRunJobDialogProps> {
           }
       }
 }
+
+export async function showRunJobDialog(){
+    const result = await showDialog({
+      title: 'Run Job',
+      body: (
+        <RunJobDialog
+          jobPath={sessionStorage.getItem('current-path')!}
+        ></RunJobDialog>
+      ),
+      buttons: [Dialog.okButton(), Dialog.cancelButton()]
+    });
+    const resultButtonClicked = !result.value && result.button.accept;
+    if (resultButtonClicked) {
+      let response = await jobRunRequest();
+      if(response[1]){
+        alert(response[0]);
```

Comment:
nit: may add TODO so we don't forget a dev alert somewhere (the IDEs have support for enumerating TODOs)